fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -295,11 +277,14 @@ private RefreshJobStatus doMvRefresh(TaskRunContext context, MaterializedViewMet
             }
         }
 
+        // Unlock current database and acquire lock for all databases referenced by the plan
+        insertStmt = prepareRefreshPlan(refTableRefreshPartitions, mvToRefreshedPartitions, refTablePartitionNames);
+
```

Comment:
Is there a chance that  base or MV's metadata(schema or partition strategy) changing is wedged between getRefTableRefreshPartitions and prepareRefreshPlan? 
so we need validate insert plan.

Reply:
The MV's meta would only be updated by this thread(RefreshProcessor), so my suspect is it would not be updated after releasing the lock.